test/hack/resource/pkg/resourcetypes/stack.go

```diff
@@ -42,80 +42,51 @@ func (s *Stack) Global() bool {
 }
 
 func (s *Stack) GetExpired(ctx context.Context, expirationTime time.Time, excludedClusters []string) (names []string, err error) {
-	var nextToken *string
-	for {
-		out, err := s.cloudFormationClient.DescribeStacks(ctx, &cloudformation.DescribeStacksInput{
-			NextToken: nextToken,
-		})
-		if err != nil {
-			return names, err
-		}
+	stacks, err := s.getAllStacks(ctx, &cloudformation.DescribeStacksInput{})
```

Comment:
nit, since we don't filter, do we need to add an Input?
```suggestion
	stacks, err := s.getAllStacks(ctx)
```

Reply:
This is related a bit to my design decision from above, from your question about passing in `Filter` instead of `Input`. I could wrap the empty `Input` into the getAllStacks function, since no other function in `stack.go` passes in a non-empty `Input`, I was otherwise just maintaining consistency with the other `getAll<Resource>` functions.

Reply:
I think we can drop it in this context. We should only pass something if we are going to use it. It would be easy to add back in later.